### PR TITLE
zh: add doc about suspending

### LINKED ISSUE #1985: Add doc about suspending cluster
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed, added, or deleted? (Required)
Close #1985 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/suspend-tidb-cluster.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: 挂起 TiDB 集群
+summary: 介绍如何通过配置挂起 Kubernetes 上的 TiDB 集群
+---
+
+本文介绍如何通过配置 `TidbCluster` 对象来挂起 Kubernetes 上的 TiDB 集群。挂起集群指的是：保留 `TidbCluster` 对象，并停止所有组件或者某个组件的 Pod。
+
+在某些测试场景下，你可以在不使用 TiDB 集群时挂起集群，并且可以移除 TiDB 集群所占用的 Node 或者 Service，以此来节省成本。等待再次使用时恢复 TiDB 集群。
+
+目前，挂起 TiDB 集群只支持删除组件的 Pod，其他资源（例如 Service、PVC 等）都会被保留。
```
Comment (csuzhangxc) [inline zh/suspend-tidb-cluster.md:10]: L8 说可以移除 Service，这里说不会删除，要更多的解释吗

File: zh/suspend-tidb-cluster.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    metadata:
+      name: ${cluster_name}
+      namespace: ${namespace}
+    spec:
+      suspendAction:
+        suspendStatefulSet: false
+      # ...
+    ```
+  
+    TiDB Operator 也支持恢复一个或多个 TiDB 集群的组件。以 TiKV 为例，通过配置 `TidbCluster` 对象的 `spec.tikv.suspendAction` 字段来挂起 TiDB 集群中的 TiKV。
```
Comment (csuzhangxc) [inline zh/suspend-tidb-cluster.md:76]:
```suggestion
    TiDB Operator 也支持恢复一个或多个 TiDB 集群的组件。以 TiKV 为例，通过配置 `TidbCluster` 对象的 `spec.tikv.suspendAction` 字段来恢复 TiDB 集群中的 TiKV。
```

**Comment by ran-huang** [comment]
这个适用哪些版本的？

File: zh/suspend-tidb-cluster.md (hunk 1 above)
Comment (KanShiori) [reply zh/suspend-tidb-cluster.md]: Done，改为直接说节省资源了

**Comment by KanShiori** [comment]
> 这个适用哪些版本的？

>= 1.3.7，加了版本要求

File: zh/suspend-tidb-cluster.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      tikv:
+        suspendAction:
+          suspendStatefulSet: false
+      # ...
+    ```
+
+2. 恢复 TiDB 集群后，通过以下命令观察到挂起的组件的 Pod 逐步被创建。
+
+    ```shell
+    kubectl -n ${namespace} get pods
```
Comment (ran-huang) [inline zh/suspend-tidb-cluster.md:100]: 各个组件 Pod 恢复的顺序也写一下？

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: zh/suspend-tidb-cluster.md (hunk 3 above)
Comment (KanShiori) [reply zh/suspend-tidb-cluster.md:100]: 恢复没有规定顺序，就跟部署一样。感觉不用写

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**